NESTED.md

```diff
@@ -0,0 +1,204 @@
+\# Nested Execution
+
+## Approving the transaction
+
+### 1. Update repo:
+
+```
+cd superchain-ops
+git pull
+just install
+```
```

Comment:
Unlike `SINGLE.md`, this one has no info about `cd`'ing to the appropriate folder

Reply:
Must be an error in SINGLE, you shouldn't need to `cd` anymore, you just need to specify the path to the right task. 

But maybe it would be better to `cd` and simplify the invocation. That would require passing in the path to the justfile in the root, I'll have to confirm that works.